src-built-in/components/myApps/src/stores/StoreActions.js

```diff
@@ -95,13 +95,41 @@ function getApp(appID, cb = Function.prototype) {
 }
 // Check to see if an app is already in our list of apps
 function appInAppList(appName) {
-	let components = Object.values(data.apps);
-	for (let i = 0; i < components.length; i++) {
-		let component = components[i];
-		if (component.name === appName) return true;
+	let app = findAppByField('name', appName);
+	return Boolean(app);
```

Comment:
Unnecessary variable `app`
```suggestion
	return findAppByField('name', appName);
```